templates/user/auth/signup_inner.tmpl

```diff
@@ -54,6 +54,14 @@
 						<div class="h-captcha" data-sitekey="{{ .HcaptchaSitekey }}"></div>
 					</div>
 				{{end}}
+				{{if and .EnableCaptcha (eq .CaptchaType "mcaptcha")}}
+					<div class="inline field required df ac">
+						<label></label>
+						<div class="border-secondary" id="mcaptcha__widget-container" style="width: 304px; height: 78px"></div>
```

Comment:
Can we avoid these specific width/height values?

Reply:
mCaptcha's widget expect these dimensions(without these, it will not look centered)
![image](https://user-images.githubusercontent.com/25481501/181093522-91dead6e-2681-4b9d-8f45-9be9ef31ab4c.png)

Reply:
Hmm okay. I have to say it looks a bit amateurish with non-vertically-centered content in the box and that big icon. Are you sure it's integrated correctly?

Reply:
> I have to say it looks a bit amateurish with non-vertically-centered content in the box

That's without the explicit width/height, to show that it won't work. It actually looks like this:
![image](https://user-images.githubusercontent.com/25481501/181110146-f15b7cb9-cd83-4b81-96d3-1b2a20f50c5f.png)



> that big icon.

🤷🏽 it's what mCaptcha provides. 

> Are you sure it's integrated correctly?

Yes(*kind of*), but I'm working on mCaptcha to use better CSS, as currently it's broken for mobile.

Reply:
> That's without the explicit width/height, to show that it won't work

That looks okay to me, at least from a vertical centering standpoint.